screens/room/AddRoomScreen.js

```diff
@@ -1,14 +1,289 @@
-import React from 'react';
-import {View, Text, StyleSheet} from 'react-native';
+import React, { useState, Component } from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  Image,
+  ScrollView,
+  TextInput
+} from "react-native";
 
-const AddRoomScreen = () =>{
-    return(
-        <View></View>
+import ToggleSwitch from "toggle-switch-react-native";
+import NHSInputAndroid from "../../components/NHSInputAndroid";
+// Expo Vector-Icons
+import {
+  FontAwesome,
+  EvilIcons,
+  MaterialCommunityIcons,
+  Foundation
+} from "@expo/vector-icons";
+
+export class AddRoomScreen extends Component {
```

Comment:
We've used functional components so far, can you update this one as well?

https://reactjs.org/docs/hooks-intro.html#motivation
https://stackoverflow.com/questions/36097965/when-to-use-es6-class-based-react-components-vs-functional-es6-react-components